.github/config/android-arm.json

```diff
@@ -13,9 +13,7 @@
         "gpu_unittests",
         "gin_unittests",
         "blink_unittests",
-        "content_shell",
```

Comment:
atleast for this builder and the other non-cobalt ones , the idea is to build chromium important targets with non-cobalt changes. So it doesn't make sense to add cobalt_apk here IMO(it's not a chromium target)

Reply:
+1, I think you meant to add them to the cobalt-specific json file